backend/json.go

```diff
@@ -0,0 +1,93 @@
+package backend
+
+import (
+	jsoniter "github.com/json-iterator/go"
+)
+
+// // This will make sure jsoniter uses a fast JSON serialization strategy
+// var (
+// 	_ = initEncoders()
```

Comment:
@aknuds1 any thoughts on registering these codecs?

@FZambia is nervous about it... it does seem like a good way to get the best performance out of our serialization stack  (OBVIOUSLY other importnat things to fix also!)

Reply:
Benchmarked `data.Frame` encoding to be aware of impact:
```
BenchmarkFrameToJSONFunction-12                        	   44614	     25736 ns/op	    2955 B/op	     148 allocs/op
BenchmarkFrameMarshalStdLib-12                         	   20386	     58279 ns/op	    7857 B/op	     149 allocs/op
BenchmarkFrameMarshalJsonIter-12                       	   43723	     27351 ns/op	    7862 B/op	     150 allocs/op
BenchmarkFrameMarshalJsonIterWithEncoderRegistry-12    	   45954	     26159 ns/op	    7478 B/op	     139 allocs/op
```